README.md

```diff
@@ -43,18 +43,43 @@ addressing a single deprecation at a time, and prevents backsliding
 
 ### Getting started
 
-The initial steps needed to get started:
+#### Normal setup routine
```

Comment:
I think we should only communicate one setup routine, if the embroider one works fine for classic let's just do that

Reply:
Done.